apps/dashboard/src/contract-ui/hooks/useRouteConfig.tsx

```diff
@@ -383,23 +297,32 @@ export function useContractRouteConfig(
     {
       title: "NFTs",
       path: "nfts",
-      isEnabled: contractData.detectedNftExtensions,
+      isEnabled:
+        isERC721Query.data || isERC1155Query.data
+          ? "enabled"
+          : isERC721Query.isLoading || isERC1155Query.isLoading
+            ? "loading"
+            : "disabled",
```

Comment:
cc @kien-ngo - check out the new extension detection pattern, might need your help completing the work there to migrate the rest of it